File: seekseek/lib/entity/reply.dart
class replyEntity {
  int _replyId;
  int _replyParent;
  int _replyParentType;
  int _replyCreator;
  String _replyCreatorName;
  String _replyContent;
  String _replyCtime;
  int _replyLikes;
  int _replyDislikes;

  replyEntity(
      this._replyId,
      this._replyParent,
      this._replyParentType,
      this._replyCreator,
      this._replyCreatorName,
      this._replyContent,
      this._replyCtime,
      this._replyLikes,
      this._replyDislikes);

  int get replyId => _replyId;

  set replyId(int value) {
    _replyId = value;
  }

  int get replyParent => _replyParent;

  set replyParent(int value) {
    _replyParent = value;
  }

  int get replyParentType => _replyParentType;

  set replyParentType(int value) {
    _replyParentType = value;
  }

  int get replyCreator => _replyCreator;

  set replyCreator(int value) {
    _replyCreator = value;
  }

  String get replyCreatorName => _replyCreatorName;

  set replyCreatorName(String value) {
    _replyCreatorName = value;
  }

  String get replyContent => _replyContent;

  set replyContent(String value) {
    _replyContent = value;
  }

  String get replyCtime => _replyCtime;

  set replyCtime(String value) {
    _replyCtime = value;
  }

  int get replyLikes => _replyLikes;

  set replyLikes(int value) {
    _replyLikes = value;
  }

  int get replyDislikes => _replyDislikes;

  set replyDislikes(int value) {
    _replyDislikes = value;
  }
}

File: xunmi/src/main/java/com/tutulei/xunmi/bean/Reply.java
package com.tutulei.xunmi.bean;

import java.sql.Timestamp;

public class Reply {
    private int replyId;
    private Integer replyParent;
    private Byte replyParentType;
    private Integer replyCreator;
    private String replyCreatorName;
    private String replyContent;
    private Timestamp replyCtime;
    private Integer replyLikes;
    private Integer replyDislikes;

    public String getReplyCreatorName() {
        return replyCreatorName;
    }

    public void setReplyCreatorName(String replyCreatorName) {
        this.replyCreatorName = replyCreatorName;
    }

    public int getReplyId() {
        return replyId;
    }

    public void setReplyId(int replyId) {
        this.replyId = replyId;
    }

    public Integer getReplyParent() {
        return replyParent;
    }

    public void setReplyParent(Integer replyParent) {
        this.replyParent = replyParent;
    }

    public Byte getReplyParentType() {
        return replyParentType;
    }

    public void setReplyParentType(Byte replyParentType) {
        this.replyParentType = replyParentType;
    }

    public Integer getReplyCreator() {
        return replyCreator;
    }

    public void setReplyCreator(Integer replyCreator) {
        this.replyCreator = replyCreator;
    }

    public String getReplyContent() {
        return replyContent;
    }

    public void setReplyContent(String replyContent) {
        this.replyContent = replyContent;
    }

    public Timestamp getReplyCtime() {
        return replyCtime;
    }

    public void setReplyCtime(Timestamp replyCtime) {
        this.replyCtime = replyCtime;
    }

    public Integer getReplyLikes() {
        return replyLikes;
    }

    public void setReplyLikes(Integer replyLikes) {
        this.replyLikes = replyLikes;
    }

    public Integer getReplyDislikes() {
        return replyDislikes;
    }

    public void setReplyDislikes(Integer replyDislikes) {
        this.replyDislikes = replyDislikes;
    }
}
